src/app/src/main/kotlin/org/imperial/mrc/hint/AppStart.kt

```diff
@@ -35,4 +33,15 @@ class MvcConfig(val config: Config) : WebMvcConfigurer {
                 .addPathPatterns("/**")
                 .excludePathPatterns("/login", "/login/", "/password/**", "/callback", "/callback/", "/public/**")
     }
+
```

Comment:
Spring requires this to be configured for StreamingResponseBody. Settings might need tweaking.